store.go

```diff
@@ -0,0 +1,141 @@
+package cbornode
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	block "github.com/ipfs/go-block-format"
+	cid "github.com/ipfs/go-cid"
+	mh "github.com/multiformats/go-multihash"
+	recbor "github.com/polydawn/refmt/cbor"
+	atlas "github.com/polydawn/refmt/obj/atlas"
+	cbg "github.com/whyrusleeping/cbor-gen"
+)
+
+type CborIpldStore interface {
```

Comment:
Name it just `IpldStore`, `cbor` is already contained in package name.

Reply:
will also rename to IpldBLockstore